include/deal.II/base/function.h

```diff
@@ -164,8 +166,9 @@ class Function : public FunctionTime<RangeNumberType>, public Subscriptor
    * (which defaults to one, i.e. a scalar function), and the time variable,
    * which defaults to zero.
    */
-  Function(const unsigned int    n_components = 1,
-           const RangeNumberType initial_time = 0.0);
+  Function(const unsigned int n_components = 1,
+           const typename numbers::NumberTraits<RangeNumberType>::real_type
```

Comment:
shall we introduce `TimeNumberType` type at the beginning of this class and use it throughout to make code easier to read?

Reply:
Yes, I think that this would be a good idea. Then users would have easy access to this type definition.